packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -95,14 +129,20 @@ class DescopeWc extends BaseDescopeWc {
         const { body } = await fetchContent(htmlLocaleUrl, 'text');
         return body;
       } catch (ex) {
-        this.logger.error(
-          `Failed to fetch html from ${htmlLocaleUrl}, error: ${ex}. Fallback to url ${htmlUrl}`
+        this.logger().error(
+          `Failed to fetch html page from ${htmlLocaleUrl}. Fallback to url ${htmlUrl}`,
+          ex
         );
       }
     }
 
-    const { body } = await fetchContent(htmlUrl, 'text');
-    return body;
+    try {
+      const { body } = await fetchContent(htmlUrl, 'text');
+      return body;
+    } catch (ex) {
+      this.logger().error(`Failed to fetch html page from ${htmlUrl}`, ex);
```

Comment:
Isn't it redundant?
We are already logging an error on the fetchContent fn

Reply:
we are not logging, we are throwing. this will log it with context, similar to other fetches.